feature/welcome/src/main/java/com/rahim/yadino/welcome/WelcomeViewModel.kt

```diff
@@ -19,8 +18,8 @@ class WelcomeViewModel @Inject constructor(
 ) : ViewModel(), WelcomeContract {
 
   private val mutableState = MutableStateFlow<WelcomeContract.WelcomeState>(WelcomeContract.WelcomeState())
-  override val state: StateFlow<WelcomeContract.WelcomeState> = mutableState.onStart {
-    isShowWelcomeScreen()
+  override val state: StateFlow<WelcomeContract.WelcomeState> = isShowWelcomeScreen().map {
+    WelcomeContract.WelcomeState(it)
```

Comment:
why not use mutableState??!!

please explain to me

Reply:
reverted